### PR TITLE
[Doc][Kommander] 1.4 GA docs

### PR DESCRIPTION
**Pull request by russurquhart1**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-75019
## Description of changes being made
Kommander 1.4. Release Notes

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `main`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 1]
```diff
@@ -30,6 +30,11 @@ Rate limiting happens on a per-pull basis regardless of whether the pulled image
 
 For more information on addressing this limit, see [Docker hub rate limits](../operations/manage-docker-hub-rate-limits).
 
+## New features
```
Comment (cmastr) [inline pages/dkp/kommander/1.4/release-notes/index.md:33]: @russurquhart1 this section needs to be reformatted to look like this:

https://docs.d2iq.com/dkp/kommander/1.3/release-notes/1.3/#new-features

**Review by cmastr — CHANGES_REQUESTED** [review]

File: pages/dkp/kommander/1.4/clusters/attach-cluster/cluster-no-network-restrictions/index.md [hunk 2]
```diff
@@ -21,4 +21,4 @@ Use this option when you want to attach a cluster that does not require addition
 
 1. Select the **Submit** button to begin the cluster attachment processing.
 
-![Add Cluster Connect](/dkp/kommander/latest/img/add-cluster-connect.png)
\ No newline at end of file
+![Add Cluster Connect](/dkp/kommander/latest/img/add-cluster-connect.png)
```
Comment (msdolbey) [inline pages/dkp/kommander/1.4/clusters/attach-cluster/cluster-no-network-restrictions/index.md:24]:
```suggestion
![Add Cluster Connect](/dkp/kommander/1.4/img/add-cluster-connect.png)
```

File: pages/dkp/kommander/1.4/clusters/attach-cluster/cluster-no-network-restrictions/index.md [hunk 3]
```diff
@@ -21,4 +21,4 @@ Use this option when you want to attach a cluster that does not require addition
 
 1. Select the **Submit** button to begin the cluster attachment processing.
 
-![Add Cluster Connect](/dkp/kommander/latest/img/add-cluster-connect.png)
\ No newline at end of file
+![Add Cluster Connect](/dkp/kommander/1.4/img/add-cluster-connect.png)
```
Comment (msdolbey) [inline pages/dkp/kommander/1.4/clusters/attach-cluster/cluster-no-network-restrictions/index.md:24]: edited this - it should almost always be versioned for images.

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 4]
```diff
@@ -30,6 +30,11 @@ Rate limiting happens on a per-pull basis regardless of whether the pulled image
...
+
+- The Projects tab can now show all of the current Helm Release charts, their chart version, and the names of the clusters.
+- Kommander now supports attaching existing, external clusters that have additional networking restrictions in place such as a DMZ, a proxy server, or being behind a firewall. More information is available at, [Attaching an Existing Kubernetes Cluster](https://docs.d2iq.com/dkp/kommander/1.4/clusters/attach-cluster/).
+
 ## Component versions
```
Comment (msdolbey) [inline pages/dkp/kommander/1.4/release-notes/index.md:38]: We'll have to update these versions, too. I'm not sure where they all come from - but if you need me to check I can get them all.

**Comment by russurquhart1** [comment]
@cmastr or @msdolbey I can find what the conflict is here. Can you guys see it?

File: pages/dkp/kommander/1.4/release-notes/index.md (hunk 4 above)
Comment (russurquhart1) [reply pages/dkp/kommander/1.4/release-notes/index.md:41]: Yes please!

**Comment by russurquhart1** [comment]
I think i figured it out!

**Review by cneth — APPROVED** [review]
LGTM, once Casie and Dolbey's comments are addressed!

**Pull request closed without merging**